### PR TITLE
update benchmark table in readme

### LINKED ISSUE #69: update benchmark table in readme
Please could the benchmark table be updated in the readme.

It's been a while and it would be nice to include the new frameworks.

Thanks!

### PR DESCRIPTION
**Pull request by JamesKyburz**
I ran the benchmarks twice as done previously.

This closes #69.

### CODE REVIEW

**Comment by StarpTech** [comment]
Please run it on the current node.js LTS (8.12.0)

**Comment by JamesKyburz** [comment]
@StarpTech OK, was unsure because node 10 is days away ;) 

Running the benchmarks as we speak with 8.12.0.

File: README.md [hunk 1]
```diff
@@ -40,56 +40,68 @@ benchmark [arguments (optional)]
 
 # Benchmarks
 
-* __Machine:__ MacBook Pro (Late 2013 | 2.3 GHz Intel Core i7 | 16 GB 1600 MHz DDR3)
+* __Machine:__ MacBook Pro (Late 2016 | 2,7 GHz Intel Core i7 | 16 GB 2133 MHz LPDDR3)
 * __Method:__ `autocannon -c 100 -d 5 -p 10 localhost:3000` (two rounds; one warm-up, one to measure).
-* __Node:__ `8.11.2`
+* __Node:__ `10.12.0`
```
Comment (cemremengu) [inline README.md]: `8.12.0`

File: README.md [hunk 2]
```diff
@@ -40,56 +40,68 @@ benchmark [arguments (optional)]
...
 
 ```
 ┌──────────────────────────┬─────────┬────────┬────────────┬─────────┬───────────────┐
 │                          │ Version │ Router │ Requests/s │ Latency │ Throughput/Mb │
 ├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ bare                     │ 8.11.2  │ ✗      │ 30156.8    │ 3.23    │ 4.36          │
+│ bare                     │ 8.11.2  │ ✗      │ 38224      │ 2.54    │ 5.38          │
```
Comment (cemremengu) [inline README.md]: `8.12.0`

**Comment by JamesKyburz** [comment]
Updated `PR` readme + bare version as `8.12.0`

**Comment by aichholzer** [comment]
Thank you @JamesKyburz

**Pull request merged**